src/Shared/Recruit.Shared.Web/Extensions/EnumExtensions.cs

```diff
@@ -17,6 +17,12 @@ public static string GetDisplayName(this Enum enumValue)
             return displayName ?? enumValue.ToString();
         }
 
+        public static bool IsInLiveVacancyOptions(this Enum enumValue)
```

Comment:
Can the extension be applied on the specific type instead of generic Enum that way you don't have to use ToString and compare direct values.

Reply:
Also add some tests on the extension method.